conf/bi.config

```diff
@@ -4,17 +4,12 @@ params{
   config_profile_url = 'https://www.boehringer-ingelheim.com/'
 }
 
-params.globalConfig = set_global_config()
-
-
-def set_global_config() {
-  def config = System.getenv('NXF_GLOBAL_CONFIG')
-  if(config  == null)
-  {
-      def errorMessage = "WARNING: For bi.config requires NXF_GLOBAL_CONFIG env var to be set. Point it to global.config file if you want to use this profile."
-      System.err.println(errorMessage)
-  }else{
-    includeConfig config
-  }
-  return config
+def configVar = System.getenv('NXF_GLOBAL_CONFIG')
```

Comment:
Can you just set this as `params.globalConfig` here directly?

Reply:
Sure, just pushed it.